development/README.md

```diff
@@ -282,4 +282,191 @@ $BUILD_SCRIPTS/qgis-dev-install.sh $BUILD_DIR
 
 ## Configure/build/install QGIS in Qt Creator.app
 
-TODO
+### Install QtCreator 4.0 or better
+
+* Grab the installer from https://www.qt.io/download-open-source/
+* Run the installer. You ONLY need the qt-creator package so disable the Qt5 and don’t install Qt libs (we will use the qt4 installed by brew)
+* Once the installer is run open qt-creator
+* Make a directory in your QGIS source tree called ``build``
```

Comment:
@m-kuhn Personally I just override it so it always builds into ./QGIS/build - I don't see much use for building different profiles and the very long directory names that QtCreator generates. I will update the notes below to remove the relwithdebinfo bit.